PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -810,6 +810,16 @@ struct HfCandidateCreator2ProngExpressions {
     }
   }
 
+  void processMc(aod::TracksWMc const& tracks,
+                 aod::McParticles const& mcParticles,
+                 McCollisionsFT0Cs const& collInfos,
+                 aod::McCollisions const& mcCollisions,
+                 BCsInfo const& BCsInfo)
+  {
+    runCreator2ProngMc<CentralityEstimator::None>(tracks, mcParticles, collInfos, mcCollisions, BCsInfo);
+  }
+  PROCESS_SWITCH(HfCandidateCreator2ProngExpressions, processMc, "Process MC - no centrality", false);
```

Comment:
```suggestion
  PROCESS_SWITCH(HfCandidateCreator2ProngExpressions, processMc, "Process MC - no centrality", true);
```
Please put the others false by default. Also add a protection in the `init` to forbid the users to enable more than one process function at a time (as done in the reco part). 

This comment holds for all the creators, thanks!